specification/app/resource-manager/Microsoft.App/preview/2024-10-02-preview/examples/ContainerApps_Kind_CreateOrUpdate.json

```diff
@@ -97,7 +97,8 @@
                 "resources": {
                   "cpu": 1.0,
                   "memory": "2Gi",
-                  "ephemeralStorage": "4Gi"
+                  "ephemeralStorage": "4Gi",
+                  "gpu": 1.0
```

Comment:
Could we update example for init container property and jobs?

Reply:
Go ahead please.

Reply:
Updated examples